whois-commons/src/main/java/net/ripe/db/whois/common/sso/AuthServiceClient.java

```diff
@@ -250,4 +252,21 @@ public List<String> getOrgsContactsEmails(final Long membershipId) {
 
         return Collections.emptyList();
     }
+
+    @Nullable
+    public MemberContactsResponse getOrgsContactsGroupsEmails(final Long membershipId) {
+        // Used by whois-internal. Do not remove
+        if (membershipId == null){
+            return null;
+        }
+
+        return client.target(restUrl)
+                    .path(ORGANISATION_MEMBERS_PATH)
+                    .path(String.valueOf(membershipId))
+                    .path(ACCOUNTS_PATH)
+                    .request(MediaType.APPLICATION_JSON_TYPE)
+                    .header(API_KEY, apiKey)
+                    .get(MemberContactsResponse.class);
```

Comment:
Catch all exceptions like the above method and it should throw AuthServiceClientException?